next-on-netlify.js

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env node
 const { program } = require("commander");
 
-program
-  .option(
-    "--max-log-lines [number]",
-    "lines of build output to show for each section",
-    50
-  )
-  .parse(process.argv);
-
 const nextOnNetlify = require("./index");
 const { logTitle } = require("./lib/helpers/logger");
 
-nextOnNetlify();
+program.option(
+  "--max-log-lines [number]",
+  "lines of build output to show for each section",
+  50
+);
+
+program
```

Comment:
We can use an option (e.g. `--watch`) instead of a command, but with a command we can add specific watch configuration (e.g. `watch --path a --path b`)